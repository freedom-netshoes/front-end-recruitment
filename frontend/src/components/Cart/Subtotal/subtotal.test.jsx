/* global describe, it, expect */
import React from 'react'
import SubTotal from './sub-total'
import { shallow } from 'enzyme'

describe('Testing <SubTotal /> component', () => {
  it('renders the SubTotal component', () => {
    const component = shallow(<SubTotal />)
    expect(component).toHaveLength(1)
  })

  it('Testing <Subtotal /> snapshop', () => {
    const component = shallow(<SubTotal {...{
      priceTotal: 795.22,
      maxInstallment: 5
    }} />)
    expect(component).toMatchSnapshot()
  })
})
