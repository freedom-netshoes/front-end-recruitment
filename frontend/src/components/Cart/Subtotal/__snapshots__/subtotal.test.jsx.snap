// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing <SubTotal /> component Testing <Subtotal /> snapshop 1`] = `
<div
  className="row sub-total"
>
  <p
    className="col-xs-12 col-sm-6 start-sm subtotal"
  >
    Subtotal
  </p>
  <div
    className="col-xs-12 col-sm-6 end-sm"
  >
    <div
      className="subtotal-price"
    >
      <Price
        type="separated"
        value={795.22}
      />
    </div>
    <div
      className="subtotal-installments"
    >
      <Installments
        installments={5}
        type="complete"
        value={795.22}
      />
    </div>
  </div>
  <div
    className="col-xs-12 btn-buy-container"
  >
    <button
      className="btn-buy"
    >
      Comprar
    </button>
  </div>
</div>
`;
